conans/paths/package_layouts/__init__.py

```diff
@@ -0,0 +1,4 @@
+# coding=utf-8
+
+from .package_cache_layout import PackageCacheLayout
```

Comment:
Why not absolute imports as elsewhere in the conan codebase?
I mean. Most ``__init__.py`` should be blank, unless something really special is necessary.

Why cant consumers just 
``from conans.paths.layouts.package_cache_layout import PackageCacheLayout``? 
instead of 
``from conans.paths.layouts import PackageCacheLayout``?

I also prefer the ``layouts`` package instead of ``package_layouts``. We could name ``package`` most of the methods and attributes in the code base 😅 

I would leave this file blank.

Reply:
I like the way it looks when I import them 🙄 

```python
from conans.paths.package_layouts import PackageCacheLayout, PackageEditableLayout
```

and the relative import..., I'll change them (there are some more in the codebase).

about the `layouts` vs `package_layouts`, I prefer to be explicit. If `package_` is not the word, let's find a diferent one, but for me _layout_ can refer to more things (layout of a repo, of a library, of a package, also the layout of a conanfile itself).